docs/api.rst

```diff
@@ -330,6 +330,31 @@ Generates Facebook login URL to request access token and permissions.
     fb_login_url = graph.auth_url(app_id, canvas_url, perms)
     print(fb_login_url)
 
+get_code_from_token
```

Comment:
Since not every method is documented, this file has gotten a bit confusing. In general, though, methods should appear in the same order that they appear in the code (so this should be between `delete_object` and `auth_url`).